curiouslearning/assessment-survey-js
# Fixed Ui Issues for Loading Screen Occuring due to tapping on Screen!

# Changes
- Added a condition to check UIController.getInstance().contentLoaded before proceeding to the main assessment screen.
- This ensures that both the loading screen and splash screen are not skipped if the content is not fully loaded.
# How to test
- Test method 1:
 - Download an assessment.
 - Reopen the assessment and observe the loading screen.
 - Tap the screen during the loading phase.
 - If the content is fully loaded (UIController.getInstance().contentLoaded is true), the app proceeds to the assessment 
    main screen. Otherwise, the loading screen remains visible until the content is fully loaded.

Ref: Issue Mentionned in Team Type Ninjas Slack Channel by Ashish!
